targets/xiwi

```diff
@@ -28,8 +28,27 @@ if [ "${DISTROAKA:-"$DISTRO"}" = 'debian' ]; then
     apt-mark unhold xserver-xorg-video-dummy$ltspackages || true 2>/dev/null
```

Comment:
Is there an equivalent to apt-mark hold to prevent portage from recompiling dummy without the patches?